packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
-
-4. Run `ls ~/.ssh` and confirm that you see `id_rsa_vetsgov` and `id_rsa_vetsgov.pub`.
-   It's normal to see several other files, as well. Seeing `id_rsa_vetsgov` and `id_rsa_vetsgov.pub`
-   means you now have your private and public (the one with the `.pub` extension) keys! Your
-   private key should never leave your computer, and it's unnecessary and inadvisable to share it
-   with anybody.
-
-#### Configure the SOCKS proxy - for internal developers
-
-Once you have a SSH public key, follow these steps:
-
-1. Create a Pull Request to the <a href="https://github.com/department-of-veterans-affairs/devops">devops repository</a> with your SSH public key.
- 	  1. Create a new branch named `add-key-X`, where X is a unique identifier for your name. Your lower-cased first/last name is fine.
-	  1. Get the contents of your `id_rsa_vetsgov.pub` by running `pbcopy < ~/.ssh/id_rsa_vetsgov.pub`
- 	  1. Create file with your email address as the name containing the contents of `id_rsa_vetsgov.pub` in https://github.com/department-of-veterans-affairs/devops/tree/master/ansible/roles/dsva-config/files/authorized_keys
- 	  1. Add your email address to the list of emails under the appropriate group (`developers:`, `devops`, etc.) in the https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/global/config.yml file. Please try to keep this list alphebetized.
- 	  1. Create a PR for this branch and comment in the #vetsgov-devops channel on the DSVA Slack.
-1. Copy down the SSH configuration that you'll need locally to access the remote SSH servers.
- 	  1. Visit https://github.com/department-of-veterans-affairs/devops/blob/master/ssh/config
- 	  1. Click the "Raw" button, and copy the URL that's in your browser.
- 	  1. Run `curl URL > ~/.ssh/config`, where URL is the URL that you copied in the previous step. This will overwrite any existing ssh config without warning; please back up any existing file if it exists.
-1. Add your SSH key to your local agent with `ssh-add -K ~/.ssh/id_rsa_vetsgov`.
-1. If you're on OSX you can configure the ssh client to automatically add your keys with the following in `~/.ssh/config`:
-    ```
-      Host *
-        AddKeysToAgent yes
-        UseKeychain yes
-        IdentityFile ~/.ssh/id_rsa
-        IdentityFile ~/.ssh/id_rsa_vetsgov
-    ```
-1. Once your PR is merged, an update job should automatically run to update the jumpboxes and SOCKS proxy endpoints. This job takes about 5 minutes to run. Wait five minutes, then proceed to the instructions below about accessing the SOCKS proxy.
-    * If your key doesn't seem to be working, contact the Platform team in the *#vetsgov-devops* Slack channel.
+## Prerequisites and assumptions
+
+- These steps assume you're running on Linux or OSX.
+- That you have a Github account that's [configured](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account) to use `.ssh` authentication with one of your computer's public keys.
```

Comment:
Since we're pushing changes with these instructions and we're not giving https commands, this is required.